application.py

```diff
@@ -1,42 +1,54 @@
+from datetime import date
+import logging
 import os
 from PyQt5.QtWidgets import QMainWindow, QVBoxLayout, QWidget, QLabel, QGraphicsOpacityEffect
-from PyQt5.QtCore import QTimer, Qt, QPropertyAnimation
+from PyQt5.QtCore import QTimer, Qt, QPropertyAnimation,  QTime
 from PyQt5.QtGui import QPixmap
 import qrcode
 import requests
 from services import Services
 from video_player import VideoPlayer
-import logging
+from diagnostic_menu import Menu
 
 
-service = Services()
-
 BASE_URL = 'https://api.olhar.media/'
 BASE_URL_VIDEO_ENDED = 'https://api.olhar.media/?regview=1'
 ASSETS_FOLDER = "./assets"
-CURRENT_CITY = service.get_city_from_coordinates(*service.get_lat_lon())
 
+logging.basicConfig(filename=f'./assets/logs/{date.today()}.log', level=logging.DEBUG)
```

Comment:
Also, you usually don't put logs into the same directory as the application on Linux. It usually goes to `/var/log/<your-app-name>/` or somewhere similar.

Reply:
👍Moved logger into __init__ of each class instead of global scope

So I have to access `/var/log/<app>`  and store log files there? I mean, I create a `var` folder inside the active dir or use the OS `var` folder?

Reply:
Since you do not have Unix filesystem here, you do not have `/` (Unix filesystem root) to begin with. In Windows you have disks instead, so absolute paths start with `C:\`, `D:\` and such. On Unix absolute paths start with `/`. So `/` is the very root directory, which you pretty much never mess with (except when working with Docker containers, but we will open that can of worms when we get there); `/var/` top directory already exists, and I would expect `/var/log` to also exist, though I would ensure that just in case. Luckily, in Python `os.makedirs("/var/log/your-app-name", exist_ok=True)` got you covered.

Reply:
So, since you usually run it on Windows, I suggest to put an `if` when assigning your log path checking for the platform and putting a relative (and `.gitignore`d) Windows path if it runs on Windows and `/var/log/your-app-name` otherwise. Or just run VSCode on your Linux VM and always develop in Linux - that will save you a lot of hassle moving forward.